main.py

```diff
@@ -27,6 +27,13 @@ def UpdateQueue():
         api.request('statuses/retweet/:' + str(post))
         post_list.pop(0)
 
+def CheckForFollowRequest(item):
+    text = item['text']
+    if "Follow" in text:
```

Comment:
np
The word boundaries will help with not matching words that contain "follow".  There aren't many that we should actually worry about for "follow" but it's a good habit to just match what you need.  For example, we might not want to match "RT to win as long as you're not **follow**ing us!"
That's the best example I can come up with now lol